redisdb/datadog_checks/redisdb/redisdb.py

```diff
@@ -241,6 +243,16 @@ def _check_db(self):
             elif info_name in self.RATE_KEYS:
                 self.rate(self.RATE_KEYS[info_name], info[info_name], tags=tags)
 
+        clients_by_name = defaultdict(int)
+        clients = conn.client_list()
+        for c in clients:
+            name = c["name"]
+            if name == "":
+                name = DEFAULT_CLIENT_NAME
+            clients_by_name[name] += 1
```

Comment:
Looks like we can simplify this entire block using `Counter`:

```python
from collections import Counter

clients = conn.client_list()
clients_by_name = Counter(client["name"] or DEFAULT_CLIENT_NAME for client in clients)
```

Reply:
Nice trick!